docs/DeveloperGuide.md

```diff
@@ -9,7 +9,7 @@ title: Developer Guide
 
 ## **Acknowledgements**
 
-* {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+* AB3 for the basic structure of the application.
```

Comment:
maybe we should attach the link to AB3?

Hyperlink:
[AB3](https://github.com/se-edu/addressbook-level3)